cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -163,6 +163,7 @@ func (c *CStorVolumeController) cStorVolumeEventHandler(
 				fmt.Sprintf("failed to create cstorvolume validation "+
 					"failed on cstorvolum error: %v", err),
 			)
+			klog.Fatalf("Failed to validate volume config: %s", err.Error())
```

Comment:
adding a comment here would be good

Reply:
basically, the thought process behind this

Reply:
Done